src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -507,8 +507,7 @@ private void checkTimeoutBlacklist() throws IOException {
         if (timeout != null) {
             if (timeout > System.currentTimeMillis()) {
                 throw new IOException(Strings.apply(
-                        "Connecting to host %s with blacklistid %s is currently rejected due to connectivity issues.",
-                        request.uri().getHost(),
+                        "Connections with blacklistid %s are currently rejected due to connectivity issues.",
```

Comment:
Is this really a helpful error message? It is a bit of a trivial, obvious statement. 🤔 Who is going to read this in the case of errors? Also, "blacklistid" is not an English word, please adapt spelling, for instance "blacklist identifier"